api/client/gro_client.py

```diff
@@ -866,6 +866,8 @@ def get_df(self, show_revisions=False, index_by_series=False):
             See https://developers.gro-intelligence.com/data-point-definition.html
             If index_by_series is set, the dataframe is indexed by series.
             See https://developers.gro-intelligence.com/data-series-definition.html
+            If index_by_end_date is set, the dataframe is indexed by end date, with the 
```

Comment:
let's not combine two features in one PR. if we index by date do it in a separate pr. also in this implementation, if index_by_series is false then index_by_end_date has no effect which is confusing...